src/components/query/ListTimeSeriesQueryEditor.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { ChangeEvent, useState } from 'react';
+import { SelectableValue } from '@grafana/data';
+import { ListTimeSeriesQuery } from 'types';
+import { Input, Select } from '@grafana/ui';
+import { SitewiseQueryEditorProps } from './types';
+import { EditorField, EditorFieldGroup, EditorRow } from '@grafana/experimental';
+
+
+interface Props extends SitewiseQueryEditorProps<ListTimeSeriesQuery> {
+  newFormStylingEnabled?: boolean;
+}
+
+const timeSeriesTypes = [
+  {
+    label: 'ALL',
+    value: 'ALL',
+    description: 'All time series data',
+  },
+  {
+    label: 'ASSOCIATED',
+    value: 'ASSOCIATED',
+    description: "The time series is associated with an asset property.",
+  },
+  { label: 'DISASSOCIATED', value: 'DISASSOCIATED', description: "The time series isn't associated with any asset property." },
+];
+
+export const ListTimeSeriesQueryEditorFunction = (props: Props) => {
+
+  const [lastInput, setLastInput] = useState<string>()
+
+  const onAliasPrefixChange = (e: ChangeEvent<HTMLInputElement>) => {
+    const { onChange, query } = props;
+    setLastInput("prefix")
+    onChange({ ...query, aliasPrefix: e.target.value });
+  };
+
+  const onAssetIdChange = (e: ChangeEvent<HTMLInputElement>) => {
+    const { onChange, query } = props;
+    setLastInput("id")
+    onChange({ ...query, assetId: e.target.value });
+  };
+
+  const onTimeSeriesTypeChange = (sel: SelectableValue<string>) => {
+    const { onChange, query } = props;
+    if (sel.value === 'ALL' && lastInput === "prefix"){
+      onChange({ ...query, timeSeriesType: sel.value as 'ASSOCIATED' | 'DISASSOCIATED' | 'ALL' , assetId: undefined});
+    }
+    else if (sel.value === 'ALL' && lastInput === "id"){
+      onChange({ ...query, timeSeriesType: sel.value as 'ASSOCIATED' | 'DISASSOCIATED' | 'ALL' , aliasPrefix: undefined});
+    } else {
+      onChange({ ...query, timeSeriesType: sel.value as 'ASSOCIATED' | 'DISASSOCIATED' | 'ALL' });
+    }
+    
+  };
+
+  const { query } = props;
+
+  return (
+      <EditorRow>
+        <EditorFieldGroup>
+          <EditorField label="timeSeriesType" htmlFor="timeSeriesType" width={20}>
+            <Select
+              id="timeSeriesType"
```

Comment:
```suggestion
              inputId="timeSeriesType"
```

`<Select>` components are based on React Select which uses `inputId` instead of the `id` prop to associate a label with the input element.